File: Zestawy Zadań/C++/5 - Funkcje, rekurencja/Zestaw 2 zadania/5.3 Funkcja o prototypie char ZNAKI(char a, char b, int X).cpp
#include <iostream>

char ZNAKI(char a, char b, int X) {
    return (X > 0) ? a : b;
}

int main() {
    char U, V;
    int L;
    std::cout << "Podaj pierwszy znak: ";
    std::cin >> U;
    std::cout << "Podaj drugi znak: ";
    std::cin >> V;
    do {
        std::cout << "Podaj liczbê ca³kowit¹: ";
        std::cin >> L;
        std::cout << "Wynik: " << ZNAKI(U, V, L) << std::endl;
    } while (L != 0);
    return 0;
}

File: Zestawy Zadań/Python/5 - Funkcje, rekurencja/Zestaw 1 zadania/5.3 Funkcja o prototypie char ZNAKI(char a, char b, int X).py
def ZNAKI(a, b, X):
    return a if X > 0 else b

if __name__ == "__main__":
    U = input("Podaj pierwszy znak: ")
    V = input("Podaj drugi znak: ")
    while True:
        L = int(input("Podaj liczbę całkowitą: "))
        print("Wynik:", ZNAKI(U, V, L))
        if L == 0:
            break
